cairo/tests/ethereum/crypto/test_alt_bn128.py

```diff
@@ -242,6 +242,17 @@ def test_bnp12_add(self, cairo_run, p: BNP12, q: BNP12):
                 return
             assert cairo_run("bnp12_add", p, q) == expected
 
+        @given(p=..., n=...)
+        @settings(max_examples=30)
+        def test_bnp12_mul_by(self, cairo_run, p: BNP12, n: U384):
+            try:
+                expected = p.mul_by(int(n))
+            except OverflowError:  # fails for large points
+                with cairo_error(message="OverflowError"):  # Hint error
+                    cairo_run("bnp12_mul_by", p, U384(n))
+                return
+            assert cairo_run("bnp12_mul_by", p, U384(n)) == expected
```

Comment:
No need for casting, n is a U384
```suggestion
                    cairo_run("bnp12_mul_by", p, n)
                return
            assert cairo_run("bnp12_mul_by", p, n) == expected
```